### PR TITLE
Hotfixes for sec stuff that I missed on the first resprite.

### LINKED ISSUE #16350: HoS jacket missing textures for digi-legs
Reporting client version: 514.1585

## Round ID:
6561

## Testmerges:
- [[MODULAR] Fixes That Stupid Language Reset Bug That's Been Here Since Newprefs](https://github.com/Skyrat-SS13/Skyrat-tg/pull/15802)
- [the ash farming expands](https://github.com/Skyrat-SS13/Skyrat-tg/pull/15898)
- [[SEMI-MODULAR] A (Nearly) complete rework and rebalance of xenos](https://github.com/Skyrat-SS13/Skyrat-tg/pull/15863)
- [Revert "[MIRROR] Add Alt RMB and Ctrl RMB screentips (used for Simple Rotation) [MDB IGNORE]"](https://github.com/Skyrat-SS13/Skyrat-tg/pull/16298)
- [Adds several themed vending machines that can spawn in place of snack vendors](https://github.com/Skyrat-SS13/Skyrat-tg/pull/16215)
- [Izhevsk Coalitioning](https://github.com/Skyrat-SS13/Skyrat-tg/pull/15696)

## Reproduction:
If you have digitigrade legs, every Head of Security jacket variant except the armored trenchcloak has a missing texture, resulting in only a single variant being useable without a horrible pink and black eyesore covering your character.

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request
Fixes some stuff that were forgotten and just in general. Straight jacket now actually paths properly, secmed gear now has digi variations, the blue secmed labcoat now has mob sprites.

Bulletproof helm is the version before the redsec resprite to keep the original visor.
HoS jacket issue on digitigrade has also been fixed.

Fixes #16350

## How This Contributes To The Skyrat Roleplay Experience
Fixes.

## Changelog

:cl:
fix: Security medic's clothing now fit on digitigrade legs.
fix: Security medic's mysterious error straight jackets are now actual straight jackets.
fix: HoS' trenchcoats are no longer missing textures for digitigrade legged HoSes
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm [hunk 1]
```diff
@@ -36,20 +36,21 @@
...
+/obj/item/clothing/under/rank/security/peacekeeper/security_medic
 	name = "security medics's turtleneck"
 	desc = "A comfy turtleneck with a white armband, denoting the wearer as a security medic."
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/security.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/security.dmi'
 	icon_state = "security_medic_turtleneck"
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION
 
 /obj/item/clothing/under/rank/security/peacekeeper/security_medic/skirt
 	name = "security medics's skirtleneck"
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm]: Dont forget to add `supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON` to the skirt
I'd have made this a Suggestion to commit it easy but mobile bad so i physically cant ;-;

File: modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm [hunk 2]
```diff
@@ -36,20 +36,21 @@
...
 	name = "security medics's uniform"
 	desc = "A lightly armored uniform worn by Nanotrasen's Asset Protection Medical Corps."
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/security.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/security.dmi'
 	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/uniform_digi.dmi'
 	icon_state = "security_medic_jumpsuit"
 
-/obj/item/clothing/under/rank/security/peacekeeper/security_medic/alternate
+/obj/item/clothing/under/rank/security/peacekeeper/security_medic
 	name = "security medics's turtleneck"
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm]: I don't think Medics's is right. Do we even use possessive 's for most other uniforms in the game? Could this just be `security medic [gear]`?
Either choice tho s's isn't right and a few items in here use that, may as well fix it while you're here

File: modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm [hunk 3]
```diff
@@ -12,7 +12,7 @@
 	worn_icon_state = "secmed_labcoat_blue"
 
 /obj/item/clothing/suit/hazardvest/security_medic
-	name = "security medic's vest"
+	name = "security medic vest"
 	desc = "A lightweight vest worn by the Security Medic."
 	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
```
Comment (ghost) [inline modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm:18]: While you're already here, would you miiiind fixing the emissives on this vest? It already has the emissive overlay set up and just needs an icon in the mob files: `"[icon_state]-emissive"`
It's easy enough to do: grab an already existing emissive sprite's opaque white, copy our icon state, remove all but the reflective material, and replace all colors of the reflective material with the opaque white. You're left with an opaque-white overlay to put atop the reflective material.

Sadly you'll need one for secmed_vest_blue too, for emissive code isn't... the best... but it'll be a copy so you can just dupe one and rename it

Comment (Ebin-Halcyon) [reply modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm:18]: Yeah i'll do it in the morning after I get back from work, along with addressing all your other reviews.

**Pull request merged**